lib/custom-logger.js

```diff
@@ -40,6 +40,7 @@ function timestampFunction() {
 
 function pinoOptions(moduleOptions) {
   const stream = moduleOptions.stream ? { stream: moduleOptions.stream } : {}
+  const customLevels = moduleOptions.customLogLevel ? { customLevels: { [moduleOptions.customLogLevel]: 35 } } : {}
```

Comment:
no hard coded magic number please, let's make it configurable together with the log level name

Reply:
If passed as options, I let the user to write directly all the custom log level configurations

Reply:
You are right, I updated it.